contribs/gmf/less/desktop.less

```diff
@@ -211,6 +211,9 @@ gmf-search {
 
 gmf-themeselector {
   width: 1.5 * @left-panel-width;
+  max-height: 1.5 * @left-panel-width;
```

Comment:
it's just to have (at max) a square div with a scrollbar. this size is arbitrary (I've checked with @ybolognini )